Changes

```diff
@@ -826,6 +826,10 @@ OCaml 4.11
 - #9695, #9702: no error when opening an alias to a missing module
   (Jacques Garrigue, report and review by Gabriel Scherer)
 
+- #9759: Typing without -principal is broken in 4.11 and trunk
```

Comment:
The message is quite scary and generic for an issue that appeared between 4.00.0 and 4.01.0 .